wit/mat.wit

```diff
@@ -2,7 +2,7 @@ package wasm:cv;
 /// mat resource is a matrix of bytes, wrapper around the cv::Mat type.
 ///
 interface mat {
-    use types.{mix-max-loc-result};
+    use types.{mix-max-loc-result, rect};
 
     enum mattype {
```

Comment:
This could be %type, which would resolve to Type in Go.

Reply:
I initially had it that way, but changed it to avoid confusion.